src/style.css

```
@@ -1 +1,40 @@
-/* add styles here */
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
... ,
.stars--5 .stars__star:nth-child(-n + 5) {
```
Consider using commas to group selectors